Clappr-tvOS/Classes/Plugin/Playback/NoOpPlayback.swift

```diff
@@ -0,0 +1,53 @@
+open class NoOpPlayback: Playback {
+    fileprivate var errorLabel = UILabel(frame: CGRect.zero)
```

Comment:
This class doesn't have too much difference between the shared one, what do you think about merging it?

Reply:
Hey @CristianDRM, I created a card to do this change in another moment.